scripts/components/chart/dinoDietChart.component.js

```diff
@@ -9,10 +10,58 @@ const data = [{
 }, {
     count: 6
 }]*/
+// generates the charts section -vinc
+const renderCharts = () =>{
+
+  const innerHTML = `
+  <div class="charts-container flex">
+    <canvas id="dietChart">
+    </canvas>
+    <canvas id="eraChart">
+    </canvas>
+  </div>
+  `;
+  const dinoDiet = calculateDiet();
+  // create a section, appends the charts-container is the parent container holding the chart
+  const section = createEle('section', innerHTML, document.querySelector('main'), "section-features", 'charts');
+  // create a new chart, where the <canvas> of id: 'dietChart'
+  new Chart(
+    document.querySelector('#dietChart'),
+    {
+      type: 'pie',
+      data: {
+        labels: ["carnivorous", "herbivorous", "omnivorous"],
+        datasets: [
+          {
+            label: 'diet',
+            data: [dinoDiet.carnivorous, dinoDiet.herbivorous, dinoDiet.omnivorous]
+          }
+        ]
+      }
+    }
+  );
+  new Chart(
```

Comment:
Temporary pie chart placed to test "flex" classList for mobile and large screens.